tests/test_replace_im_links.py

```diff
@@ -1,5 +1,8 @@
+import botocore.stub
+import boto3
 import os
-from mbtools import replace_im_links
+from mbtools import copy_resources_s3, replace_im_links
+from mbtools import fetch_im_resources
```

Comment:
Minor: You can combine these to be `from mbtools import copy_resources_s3, replace_im_links, fetch_im_resources`